src/controllers/customerController.ts

```diff
@@ -0,0 +1,15 @@
+import {getAccountsByCustomerNumber} from "../models/accountModel";
```

Comment:
If you're putting spaces around the words in the {} in other places (like the line below), make sure to do it consistently everywhere!